tldw_Server_API/app/core/Sharing/clone_service.py

```
@@ -0,0 +1,210 @@
+            # Deep copy transcripts
+            try:
+                transcripts = get_media_transcripts(self._src_media, int(media_id))
+                for t in transcripts:
+                    upsert_transcript(
+                        self._tgt_media,
+                        new_media_id,
+                        transcription=t.get("transcription", ""),
+                        whisper_model=t.get("whisper_model", "cloned"),
+                        created_at=t.get("created_at"),
+                    )
+            except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)